test/unit/org/apache/cassandra/net/MessagingServiceTest.java

```diff
@@ -361,19 +361,19 @@ public void listenPlainConnectionWithBroadcastAddr() throws InterruptedException
     @Test
     public void listenRequiredSecureConnection() throws InterruptedException
     {
-        ServerEncryptionOptions serverEncryptionOptions = new Builder().withOptional(false)
-                                                                       .withInternodeEncryption(ServerEncryptionOptions.InternodeEncryption.all)
+        ServerEncryptionOptions serverEncryptionOptions = new Builder().withInternodeEncryption(ServerEncryptionOptions.InternodeEncryption.all)
                                                                        .withLegacySslStoragePort(false)
+                                                                       .withOptional(false)
```

Comment:
@smiklosovic I assume the order changed here due to auto-formatting?

Reply:
same as above